core_lib/web_helpers/decorators.py

```diff
@@ -10,31 +10,34 @@
 logger = logging.getLogger(__name__)
 
 
-def handle_exception(func, *args, **kwargs):
+def handle_exception(func, log_exception=True,*args, **kwargs):

```

Comment:
`def handle_exception(func, log_exception=True, *args, **kwargs):`
Please add a space after log exception parameter to maintain the formatting